pallets/phala-world/src/incubation.rs

```diff
@@ -514,6 +518,10 @@ pub mod pallet {
 								("name", property_value(&sub_part_info.name)),
 								("slot_name", slot_name_value.clone()),
 								("rarity", property_value(&sub_part_info.rarity)),
+								("race", property_value(&sub_part_info.rarity)),
```

Comment:
Should be:
```suggestion
								("race", property_value(&sub_part_info.race)),
```
However, line 429-430 already add the `race` and `career` `Properties` value to the Shell parts.